demos/datagrid/script.js

```diff
@@ -13,6 +13,61 @@ require(['lib/husky'], function(Husky) {
 
     var app,
         view, pagination,
+        constants = {
+            setData: {
+                fieldList: [
+                    {
+                        "disabled": true,
+                        "name": "id"
+                    },
+                    {
+                        "translation": "public.title",
+                        "name": "title"
+                    },
+                    {
+                        "translation": "videomanagement.chapters.start",
+                        "name": "start"
+                    },
+                    {
+                        "translation": "videomanagement.chapters.end",
+                        "name": "end"
+                    }
```

Comment:
I guess you should use a more general example than `videmanagement`(because of the translations). I would go with `id`, `title` and `description`.